src/MMLib.SwaggerForOcelot/Middleware/SwaggerForOcelotMiddleware.cs

```diff
@@ -64,7 +64,18 @@ public class SwaggerForOcelotMiddleware
         public async Task Invoke(HttpContext context, ISwaggerServiceDiscoveryProvider discoveryProvider)
         {
             (string Url, SwaggerEndPointOptions EndPoint) = await GetEndPoint(context.Request.Path, discoveryProvider);
+
+            IEnumerable<ReRouteOptions> reRouteOptions = _reRoutes.Value
+                .ExpandConfig(EndPoint)
+                .GroupByPaths();
+
             HttpClient httpClient = _httpClientFactory.CreateClient();
+            string downstreamHttpVersion = reRouteOptions.FirstOrDefault()?.DownstreamHttpVersion;
```

Comment:
Please extract this lines into separate method.

Reply:
Got it.  :)